includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -2054,15 +2054,19 @@ static function() {
 
 				// Bail if the stylesheet is not enqueued.
 				if ( ! wp_style_is( $style_handle ) ) {
+					// @codeCoverageIgnoreStart
 					return;
+					// @codeCoverageIgnoreEnd
```

Comment:
Why not just do this with a single `@codeCoverageIgnore` here?

```php
if ( ! wp_style_is( $style_handle ) ) {
	return; // @codeCoverageIgnore
}
```

Reply:
Good point. Fixed in d9855a220.